gymnasium/experimental/wrappers/__init__.py

```diff
@@ -86,3 +86,38 @@
     "VectorRecordEpisodeStatistics",
     "VectorListInfo",
 ]
+
+
+class DeprecatedWrapper(ImportError):
+    """Exception raised when an old version of a wrapper is imported."""
+
+    pass
+
+
+def __getattr__(wrapper_name):
+    """Raises a DeprecatedWrapper exception when an old version of a wrapper is imported, or an ImportError if the wrapper being imported does not exist."""
+    wrapper_module = __name__
```

Comment:
We probably want to check that the wrapper name fits the expected style.

Additionally, if it doesn't match, can use `difflib` to find close matches. See https://github.com/Farama-Foundation/Gymnasium/blob/31277a8f5b66f74592ccd91873e70e21f529eb28/gymnasium/envs/registration.py#L207 for examples

Reply:
I just removed it and use plain name instead